src/Scanner.java

```diff
@@ -205,6 +207,16 @@
   }
 
   /**
+  * Specifies if the scan will be in reverse or not
+  * @param to_reverse Indication of scan direction. If this is not
+  * invoked, scanning will default to not being reversed
+  */
+  public void setReversed(boolean to_reverse){
```

Comment:
This function is only used to set reversed to true, right?  Is the to_reverse parameter necessary?  Can you pass in no parameters and have this function set is_reversed to true?